### PR TITLE
App: Support Yolov8 model 

### PR DESCRIPTION
**Pull request by niley7464**
This patch adds yolov8 ml offloading.
Tensor query client has to decode the result which server sends.

### CODE REVIEW

**Comment by niley7464** [comment]
I created yolov8 tflite model using this [guide](https://github.com/nnstreamer/nnstreamer-example/tree/main/bash_script/example_yolo#export-to-tflite-and-torchscript-model-1), but the file size is too large to push using company network 😥

**Comment by niley7464** [comment]
> I created yolov8 tflite model using this [guide](https://github.com/nnstreamer/nnstreamer-example/tree/main/bash_script/example_yolo#export-to-tflite-and-torchscript-model-1), but the file size is too large to push using company network 😥

Use ssh key and uploaded tflite mode with the help of @wooksong 😄

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt [hunk 1]
```diff
@@ -90,22 +133,29 @@ class MainActivity : ComponentActivity() {
...
-        val stop = findViewById<Button>(R.id.stop)
-        stop.setOnClickListener(View.OnClickListener {
-            mService?.stopServer()
-        })
+        // todo: Use database instead of just ArrayList
+        val modelList = ArrayList<ModelInfo>()
+        val path = getExternalFilesDir(null)!!.absolutePath
+
+        val mobileNet = File("$path/mobilenet_v1_1.0_224_quant.tflite")
+        val filter = "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```
Comment (wooksong) [inline ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt:141]: For the consistency of variable names, how about
```suggestion
        val mobinetFilter = "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```

**Comment by wooksong** [comment]
@niley7464 Please rebase this PR.

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt (hunk 1 above)
Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt]: Updated! Thank you :)

**Review by wooksong — APPROVED** [review]
LGTM

**Pull request merged**